chpl/chpl-api/e2e/collections/products-controller.postman_collection.json

```diff
@@ -1,17 +1,66 @@
 {
 	"info": {
-		"_postman_id": "6e1c03a3-35b0-496f-99c3-30c89f93aaa3",
 		"name": "products-controller",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
+	    {
+			"name": "set auth token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "e86c22fc-2510-46ff-b7f6-7db4c2add64e",
+						"exec": [
+							"var authRes = JSON.parse(responseBody);\r",
+							"if(data.KeyName = \"token\")\r",
+							"{\r",
+							"pm.environment.set(\"authToken\", authRes.token);\r",
+							"}"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					},
+					{
+						"key": "API-KEY",
+						"value": "{{apiKey}}",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\"userName\":\"{{username}}\",\"password\":\"{{password}}\"}"
+				},
+				"url": {
+					"raw": "{{url}}/auth/authenticate",
+					"host": [
+						"{{url}}"
+					],
+					"path": [
+					    "rest",
+						"auth",
+						"authenticate"
+					]
+				}
```

Comment:
This whole section of getting auth should be part of pre-request script in the request where it is getting used( In PUT request)

Reply:
That is how started with it, but env. variable wasn't picking up token from there and after a few attempts to make it work I resorted to this approach as this PR was holding off the progress on the ticket, I didn't want to delay further. That said, I'd like to see this work on bamboo and improve on it later but if it's not acceptable then I can spend some more time and try to get it working

Reply:
I have added commit making this change. Please make same change in product controller.